src/modules/Guitar/index.tsx

```diff
@@ -30,15 +30,36 @@ const Guitar = ({
     strNotes.push(getStringNotes(strings[i], instrument.frets).map(noteMapper));
   }
 
-  const fretDots = (
-    <div className={`fretDots${instrument.numberOfStrings}`}>
-      <div className="fretDot fifthFret"></div>
-      <div className="fretDot seventhFret"></div>
-      <div className="fretDot ninthFret"></div>
-      <div className="fretDot twelthFret"></div>
-      <div className="fretDot fifteenthFret"></div>
-    </div>
-  );
+
+  const fretDots = () => {
+    return instrument.numberOfStrings % 2 === 0 ? (
+      <div className={`fretDots${instrument.numberOfStrings}`}>
+        <div className="fretDot fifthFret"></div>
+        <div className="fretDot seventhFret"></div>
+        <div className="fretDot ninthFret"></div>
+        <div className="fretDot twelthFret"></div>
+        <div className="fretDot fifteenthFret"></div>
+      </div>
+    ) : (
+      <div>
+        <div className={`fretDots${instrument.numberOfStrings}`}>
+          <div className="fretDot fretDot-top fifthFret"></div>
+          <div className="fretDot fretDot-top seventhFret"></div>
+          <div className="fretDot fretDot-top ninthFret"></div>
+          <div className="fretDot fretDot-top twelthFret"></div>
+          <div className="fretDot fretDot-top fifteenthFret"></div>
+        </div>
+        <div className={`fretDots${instrument.numberOfStrings}`}>
+          <div className="fretDot fretDot-bottom fifthFret"></div>
+          <div className="fretDot fretDot-bottom seventhFret"></div>
+          <div className="fretDot fretDot-bottom ninthFret"></div>
+          <div className="fretDot fretDot-bottom twelthFret"></div>
+          <div className="fretDot fretDot-bottom fifteenthFret"></div>
+        </div>
```

Comment:
Are the above 2 blocks deliberately duplicated? Just trying to understand what is the reason for this. Would be nice to add a screenshot or two for the changes you re introducing.

Reply:
I couldn't think of a nicer way of handling this at the time. This handles the situation where there is an odd number of strings and offsets the fretDots
Changes this
![image](https://user-images.githubusercontent.com/3052698/95794875-a73a2c00-0d34-11eb-9a1a-cedb53e184e3.png)
Into this

![image](https://user-images.githubusercontent.com/3052698/95794832-8ffb3e80-0d34-11eb-8254-ff69748cdc6c.png)